### PR TITLE
irish-breakfast-shot

### PR DESCRIPTION
**Pull request by cbowersock**

### CODE REVIEW

**Comment by cbowersock** [comment]
> `irish-breakfast-shot.json` should be in `src/recipes/`

Thank you, this has been moved

File: src/recipes/irish-breakfast-shot.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        {
+            "quantity": "2oz",
+            "measure": "shot glass",
+            "ingredient": "Jameson Irish Whiskey"
+        },
+        {
+            "quantity": "1oz",
+            "measure": "shot glass",
+            "ingredient": "Orange Juice"
+        },
```
Comment (alfg) [inline src/recipes/irish-breakfast-shot.json]:
```suggestion
        }
```

File: src/recipes/irish-breakfast-shot.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    ],
+    "directions": [
+        "Pour 2oz of Jameson into a small highball glass",
+        "Add 1oz of orange juice and stir",
+        "You're done!"
+    ],
+    "image": "<irish-breakfast-shot.jpg>",
```
Comment (alfg) [inline src/recipes/irish-breakfast-shot.json]:
```suggestion
    "image": "irish-breakfast-shot.jpg",
```

File: src/recipes/irish-breakfast-shot.json (hunk 1 above)
Comment (cbowersock) [reply src/recipes/irish-breakfast-shot.json]: Fixed, thank you!

File: src/recipes/irish-breakfast-shot.json (hunk 2 above)
Comment (cbowersock) [reply src/recipes/irish-breakfast-shot.json]: Fixed, thank you!

**Comment by alfg** [comment]
Thanks for fixing!

**Pull request merged**